jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -172,11 +173,19 @@ def __init__(self, *args, **kwargs):
                                                {'name': 'calc_uncertainties', 'value': True, 'type': 'init'}]},  # noqa
             'LinearLSQFitter': {'parameters': [{'name': 'calc_uncertainties', 'value': True, 'type': 'init'}]},  # noqa
             'SLSQPLSQFitter': {'parameters': [{'name': 'maxiter', 'value': 100, 'type': 'call'}]},
-            'SimplexLSQFitter': {'parameters': [{'name': 'maxiter', 'value': 100, 'type': 'call'}]}
-        }
-        self.fitter_component = SelectPluginComponent(self, items='fitter_items',
-                                                      selected='fitter_selected',
-                                                      manual_options=list(self.all_fitters.keys()))
+            'SimplexLSQFitter': {'parameters': [{'name': 'maxiter', 'value': 100, 'type': 'call'}]},
+            'SplineSmoothingFitter' : {'parameters': [{'name': 'maxiter', 'value': 100, 'type': 'call'},  # noqa
+                                                      {'name': 's', 'value': 1.0, 'type': 'call'},
+                                                      {'name': 'degree', 'value': 3.0, 'type': 'call'},]},  # noqa
```

Comment:
It's not clear to me what "s" is in the fitter parameters, I think we either need an explanatory hint here or a more descriptive parameter name. Additionally, it's a little confusing that all the spline parameters are under the fitter - I think it would be helpful to have a note in the empty model parameters section pointing the user to the fitter.